src/Compilers/CSharp/Test/Emit/Attributes/InternalsVisibleToAndStrongNameTests.cs

```diff
@@ -1630,18 +1622,14 @@ public class C
                 var header = metadata.Module.PEReaderOpt.PEHeaders.CorHeader;
                 //confirm header has expected SN signature size
                 Assert.Equal(256, header.StrongNameSignatureDirectory.Size);
+                Assert.Equal(CorFlags.ILOnly, header.Flags);
```

Comment:
Consider adding a comment ("although a private key was provided, the assembly is still not signed" or something)